Source/JavaScriptCore/build/XCBuildData/6783d0df4ec550e8a22a7ef8ca00cffe.xcbuilddata/manifest.json

```diff
@@ -0,0 +1 @@
+{"client":{"name":"basic","version":0,"file-system":"device-agnostic","perform-ownership-analysis":"no"},"targets":{"":["<all>"]},"commands":{"<all>":{"tool":"phony","inputs":["<WorkspaceHeaderMapVFSFilesWritten>"],"outputs":["<all>"]},"P0:::Gate WorkspaceHeaderMapVFSFilesWritten":{"tool":"phony","inputs":[],"outputs":["<WorkspaceHeaderMapVFSFilesWritten>"]}}}
```

Comment:
What is the purpose of adding all these JavaScriptCore files?

Reply:
I didn't see that this was added, will remove